docker-for-mac/osxfs.md

```diff
@@ -152,3 +143,200 @@ Extended attributes are not yet supported.
 
 `osxfs` does not use OSXFUSE. `osxfs` does not run under, inside, or
 between OS X userspace processes and the OS X kernel.
+
+### Performance issues, solutions, and roadmap
+
+With regard to reported performance issues ([GitHub issue 77: File access in mounted
+volumes extremely slow](https://github.com/docker/for-mac/issues/77)), and a
+similar thread on [Docker for Mac forums on topic: File access in mounted volumes
+extremely
+slow](https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/),
+this topic provides an explanation of the issues, what we are doing about it,
+how the community can help us, and what you can expect in the future.
+This explanation is a slightly re-worked version of an [understanding
+performance
+post](https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/8076/158?u=orangesnap)
+from David Sheets (@dsheets) on the [Docker development
+team](https://forums.docker.com/groups/Docker) to the forum topic just
+mentioned. We wanted to surface it in the documentation for wider reach.
```

Comment:
> We wanted to surface it in the documentation for wider reach.

reads awkwardly but is probably fine. The text is in the present with the reader.